src/nativestore/NodeManager.cpp

```diff
@@ -332,7 +331,7 @@ NodeBlock *NodeManager::get(std::string nodeId) {
                               std::to_string(nodeBlockPointer->edgeRef));
 
     if (nodeBlockPointer->edgeRef % RelationBlock::BLOCK_SIZE != 0) {
-        throw "Exception: Invalid edge reference address = " + nodeBlockPointer->edgeRef;
+        throw std::logic_error("Exception: Invalid edge reference address = " + nodeBlockPointer->edgeRef);
```

Comment:
Lets write this to a logger instead of throwing the error

Reply:
Replaced with logger

Reply:
Ok